### PR TITLE
chore: Fix migration issue with Olympic article

### PR DESCRIPTION
**Pull request by nschonni**
Not sure I got it all, but it was mostly just adding whitespaces. Maybe the migration tool had a problem with one of the malformed markup image syntax.

### CODE REVIEW

**Comment by bariscaglar** [comment]
Figure 11 has a trailing square bracket as can be seen in "View".

**Comment by nschonni** [comment]
Sorry, I'm not seeing it in the diff or the preview. Can you comment on the specific line or push the fix to the branch?

**Comment by bariscaglar** [comment]
capture1

**Comment by bariscaglar** [comment]
capture2

**Comment by bariscaglar** [comment]
Sorry it was figure 12, not 11.

File: iis/media/live-smooth-streaming-events/nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics.md [hunk 1]
```diff
@@ -240,4 +240,74 @@ Figure 10. Workflow for ad insertion for rewind videos.
...
+![](nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics/_static/image78.jpg)
+
+Figure 11. Ad workflow for encore highlight videos. Figure 12 illustrates an example of how ad markers were incorporated in high production quality VOD encores. 
+
+[![](nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics/_static/image91.jpg)](nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics/_static/image90.jpg)]
```
Comment (nschonni) [inline iis/media/live-smooth-streaming-events/nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics.md]: Extra trailing square brace on this line

**Pull request merged**